loc/npcs/trader.js

```diff
@@ -0,0 +1,14 @@
+export default {
+  interactions: [{
+    text: [{
+      bloodlusttext: {
+        en: 'Greetings, blood-hungry warrior! What would you like to sell?'
```

Comment:
the format of all NPCs needs to be the same for now, e.g. the setup has to be like the conjurer. `text`+`choices` apart from that it looks good, you can add a choice too ("ye i want to sell items") because thats how interactions will work.

Reply:
If there is to be a positive choice for the trader, there should be a negative one also ("no thank you"), where would that go?

Reply:
there wont be "negative" choices, if you dont want to interact then you just walk away or close the dialogue box.